haystack/nodes/prompt/invocation_layer/open_ai.py

```diff
@@ -150,6 +152,10 @@ def _prepare_invoke(self, *args, **kwargs):
             "frequency_penalty": kwargs_with_defaults.get("frequency_penalty", 0),
             "logit_bias": kwargs_with_defaults.get("logit_bias", {}),
         }
+        if kwargs_with_defaults.get("response_format", None):
+            base_payload["response_format"] = kwargs_with_defaults.get("response_format")
+        if kwargs_with_defaults.get("seed", None):
+            base_payload["seed"] = kwargs_with_defaults.get("seed")
```

Comment:
I'd extract the `kwargs_with_defaults.get` call into a variable in each case and reuse it.